functions/post_form_url_encoded_test.go

```diff
@@ -36,11 +36,13 @@ const testResponse1 = `{
 }
 `
 
-const testResponse2 = `[
+const testJSONArray = `[
 "a", "b", "c"
 ]
 `
 
+const testBadResponse = `["a", "b", "c"`
```

Comment:
Sorry, I didn't think of this but we need a test to cover
1. json queries on invalid json objects `testBadJsonResponse`
1. "" queries on string response data `testStringResponse`